osf/models/node.py

```diff
@@ -871,11 +871,7 @@ def get_osf_groups_with_perms(self, permission):
         return OSFGroup.objects.filter(osfgroupgroupobjectpermission__group_id__in=member_groups)
 
     def get_aggregate_logs_query(self, auth):
-        return (
-            (
-                Q(node_id__in=list(Node.objects.get_children(self).can_view(user=auth.user, private_link=auth.private_link).values_list('id', flat=True)) + [self.id])
-            ) & Q(should_hide=False)
-        )
+        return Q(node_id=self.id) & Q(should_hide=False)
```

Comment:
Minor: Given that we'll no longer need an actual _aggregate_ logs query, should there be a follow-up maintenance ticket to refactor out much of the cruft supporting it?